docs/docs/contributing/development.mdx

```diff
@@ -631,71 +631,6 @@ To run a single test file:
 npm run test -- path/to/file.js
 ```
 
-### Integration Testing
```

Comment:
NOTE: these docs, and many other entries in these files are duplicated in `./howtos.mdx` . It's a bit tricky as they are not pure dups as there has been edits on both sides